src/tiles/grid/collection-tile.ts

```diff
@@ -43,27 +45,38 @@ export class CollectionTile extends LitElement {
     const cornerRadiusCss = css`var(--collectionTileCornerRadius, 4px)`;
 
     return css`
+      #collection-image-container {
+        /* width: 18rem; */
+        height: 18.5rem;
```

Comment:
I think you can remove this height and width with the flex updates suggested below.